kite-tools-parent/kite-tools/src/test/resources/test-schemas/tar-import.avsc

```diff
@@ -0,0 +1,14 @@
+{
+ "type": "record",
+ "name": "tarimport",
+ "fields": [
```

Comment:
This file is a duplicate of the one in main. You can reference the schema from TarFileEntry instead.